src/main/java/com/google/sps/servlets/OverTimeCasesServlet.java

```diff
@@ -0,0 +1,286 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.net.HttpURLConnection;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * Retrieves historical confirmed case data
+ * starting from 1-22-20. Takes coordinates
+ * in request and returns closest report
+ */
+@WebServlet("/timereport")
+public class OverTimeCasesServlet extends HttpServlet {
+  private HashMap<LocLatLng, ArrayList<Integer>> usTimeReports;
```

Comment:
It is good Java style to define Lists and Maps as generic as possible and then define their implementation when you actually set the list/map. For example, line 36 "should" be: "private Map<LocLatLng, List<Integer>>usTimeReports" instead of HashMap and ArrayList. The same idea can be applied lines 37-39. I lost too many points for style on cs assignments to see this and not say anything.